### PR TITLE
Exposing amp-lite in getMode and not autoplaying videos in lite mode

### PR DESCRIPTION
**Pull request by aghassemi**
/cc @rudygalfi

### CODE REVIEW

File: src/mode.js [hunk 1]
```diff
@@ -94,15 +97,17 @@ function getMode_(win) {
...
+    development: !!(hashQuery['development'] == '1' ||
         win.AMP_DEV_MODE),
     // Allows filtering validation errors by error category. For the
     // available categories, see ErrorCategory in validator/validator.proto.
-    filter: developmentQuery['filter'],
+    filter: hashQuery['filter'],
     /* global process: false */
     minified: !IS_DEV || process.env.NODE_ENV == 'production',
+    // Whether document is in an amp-lite viewer.
+    lite: searchQuery['amp_lite'] != undefined,
```
Comment (cramforce) [inline src/mode.js]: Can we be stricter on the value?

File: src/mode.js [hunk 2]
```diff
@@ -94,15 +97,17 @@ function getMode_(win) {
...
-    development: !!(developmentQuery['development'] == '1' ||
+    development: !!(hashQuery['development'] == '1' ||
         win.AMP_DEV_MODE),
     // Allows filtering validation errors by error category. For the
     // available categories, see ErrorCategory in validator/validator.proto.
-    filter: developmentQuery['filter'],
+    filter: hashQuery['filter'],
     /* global process: false */
     minified: !IS_DEV || process.env.NODE_ENV == 'production',
+    // Whether document is in an amp-lite viewer.
```
Comment (cramforce) [inline src/mode.js]: Explain what this means, as in  "Signal that the user would prefer to use less bandwidth."

Comment (aghassemi) [inline src/mode.js]: Done

File: src/mode.js (hunk 1 above)
Comment (aghassemi) [inline src/mode.js]: This is currently what lite-viewer appends to Url, a value-less `amp_lite` param on the query string.

**Pull request merged**